nexus/db-queries/src/db/datastore/mod.rs

```diff
@@ -120,6 +120,7 @@ pub use rack::RackInit;
 pub use rack::SledUnderlayAllocationResult;
 pub use region::RegionAllocationFor;
 pub use region::RegionAllocationParameters;
+pub use region_snapshot_replacement::InsertRegionSnapshotReplacementStepResult;
```

Comment:
nitpicky, take it or leave it: if this wasn't `pub use`d and we instead made the `region_snapshot_replacement` module public, then this could just be `region_snapshot_replacement::InsertStepResult` or something and code that depends on it could import it to avoid having to type out the whole thing, or not do that if it needs to be disambiguated? 

not a big deal either way though.

Reply:
I like this, done in d2135f21b